pwa/src/components/subscribers/subscriberForm.tsx

```diff
@@ -0,0 +1,513 @@
+                          id: "mappingOutAccordion",
+                          render: function () {
+                            return (
+                              <MultiDimensionalArrayInput
+                                id={"mappingOut"}
+                                label={"Mapping Out"}
+                                data={
+                                  subscriber && subscriber.mappingOut
+                                    ? [
+                                      {
+                                        key: "mappingOut",
+                                        value: `${subscriber.mappingOut}`,
+                                      },
+                                    ]
+                                    : null
+                                }
+                              />
+                            );
+                          },
+                        },
+                        {
+                          title: "Headers",
+                          id: "headersAccordion",
+                          render: function () {
+                            return (
+                              <MultiDimensionalArrayInput
+                                id={"headers"}
+                                label={"Headers"}
+                                data={
+                                  subscriber && subscriber.headers
+                                    ? [
+                                      {
+                                        key: "headers",
+                                        value: `${subscriber.headers}`,
+                                      },
+                                    ]
+                                    : null
+                                }
+                              />
+                            );
+                          },
+                        },
+                        {
+                          title: "Query Parameters",
+                          id: "queryParametersAccordion",
+                          render: function () {
+                            return (
+                              <MultiDimensionalArrayInput
+                                id={"queryParameters"}
+                                label={"Query Parameters"}
+                                data={
+                                  subscriber && subscriber.queryParameters
+                                    ? [
+                                      {
+                                        key: "queryParameters",
+                                        value: `${subscriber.queryParameters}`,
+                                      },
+                                    ]
+                                    : null
+                                }
+                              />
+                            );
+                          },
+                        },
+                      ]}
+                    />
```

Comment:
All of these fields are breaking.

![image](https://user-images.githubusercontent.com/66552932/156319507-022b008a-5e9e-4f38-80a7-c04a828bc120.png)